tsconfig.json

```diff
@@ -21,36 +21,14 @@
         "noFallthroughCasesInSwitch": true,
         "baseUrl": ".",
         "outDir": "./plugin",
+        "skipLibCheck": true,
         "paths": {
-            "nativescript-material-dialogs": ["src/dialogs/dialogs"],
-            "nativescript-material-dialogs/*": ["src/dialogs/*"],
-            "nativescript-material-snackbar": ["src/snackbar/snackbar"],
-            "nativescript-material-snackbar/*": ["src/snackbar/*"],
-            "nativescript-material-button": ["src/button/button"],
-            "nativescript-material-button/*": ["src/button/*"],
-            "nativescript-material-activityindicator": ["src/activityindicator/activityindicator"],
-            "nativescript-material-activityindicator/*": ["src/activityindicator/*"],
-            "nativescript-material-ripple": ["src/ripple/ripple"],
-            "nativescript-material-ripple/*": ["src/ripple/*"],
-            "nativescript-material-slider": ["src/slider/slider"],
-            "nativescript-material-slider/*": ["src/slider/*"],
-            "nativescript-material-progress": ["src/progress/progress"],
-            "nativescript-material-progress/*": ["src/progress/*"],
-            "nativescript-material-cardview": ["src/cardview/cardview"],
-            "nativescript-material-cardview/*": ["src/cardview/*"],
-            "nativescript-material-floatingactionbutton": ["src/floatingactionbutton/floatingactionbutton"],
-            "nativescript-material-floatingactionbutton/*": ["src/floatingactionbutton/*"],
-            "nativescript-material-core": ["src/core/core"],
-            "nativescript-material-core/*": ["src/core/*"],
-            "nativescript-material-textfield": ["src/textfield/textfield"],
-            "nativescript-material-textfield/*": ["src/textfield/*"],
```

Comment:
Ok, 
but the reason for doing this is because some packages were importing code form other packages and build them inside of them and that shouldn't happen.

you can just run the command `npm run demo` and will do all the work, but if you insist in having this approach at least I can suggest removing the pattern `/*` for all the packages like:
```
"nativescript-material-core/*": ["src/core/*"],
```
because if we keed this we are going to import code that won't be able for developers that only use the package code, and was causing some issues that were reported before because in development it will work because we have access but they only use the package code and will be a good practice to keep our development like that.

Let me know what you think

Reply:
@henrychavez ```npm run demo``` wont do everything and wont allow live change.
The idea is this:
- run the demo with ```--env.development```
- modify any ```.ts```
- you get live compilation and app restart.

Now about the actual reason why you added that. I get what you mean and i will look at it. For now let's keep it as it is